index.d.ts

```diff
@@ -0,0 +1,520 @@
+import {AllElectron, Remote, BrowserWindow, Size, Rectangle, Session, MenuItemConstructorOptions, MenuItem} from 'electron';
+import {Options as NewGithubIssueUrlOptions} from 'new-github-issue-url';
+
+/**
+ * Access the Electron APIs in both the main and renderer process without having to care about which one you're in.
+ *
+ * @example
+ *
+ * // In the renderer process:
+ * api.app.quit(); // The `app` API is usually only available in the main process.
+ */
+export const api: AllElectron | Remote;
+
+/**
+ * Check for various things.
+ */
+export const is: {
+	/**
+	 * Running on macOS.
+	 */
+	readonly macos: boolean,
+
+	/**
+	 * Running on Linux.
+	 */
+	readonly linux: boolean,
+
+	/**
+	 * Running on Windows.
+	 */
+	readonly windows: boolean,
+
+	/**
+	 * Running on the [main process](https://electronjs.org/docs/tutorial/quick-start/#main-process).
+	 */
+	readonly main: boolean,
+
+	/**
+	 * Running on the [renderer process](https://electronjs.org/docs/tutorial/quick-start/#renderer-process).
+	 */
+	readonly renderer: boolean,
+
+	/**
+	 * The app is using [ASAR](https://electronjs.org/docs/tutorial/application-packaging/).
+	 */
+	readonly usingAsar: boolean,
+
+	/**
+	 * Running in development, not in production.
+	 */
+	readonly development: boolean,
+
+	/**
+	 * The app is an Mac App Store build.
+	 */
+	readonly macAppStore: boolean,
+
+	/**
+	 * The app is a Windows Store AppX build.
+	 */
+	readonly windowsStore: boolean
+};
+
+/**
+ * Electron version.
+ *
+ * @example
+ *
+ * '1.7.9'
+ */
+export const electronVersion: string;
+
+/**
+ * Chrome version in Electron.
+ *
+ * @example
+ *
+ * '62.0.3202'
+ */
+export const chromeVersion: string;
+
+type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
+
+type RequireAtLeastOne<T, Keys extends keyof T = keyof T> =
+	Omit<T, Keys>
+	& {
+		[K in Keys]-?: Required<Pick<T, K>>
+	}[Keys]
```

Comment:
@CvX Would you be willing to submit this to https://github.com/sindresorhus/type-fest ? I'm trying to consolidate the types I use so they can be properly documented and tested in one place.